capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable no-unused-vars */
+
+/*
+ * Largely inspired by JS code from W3Schools: https://www.w3schools.com/howto/howto_js_autocomplete.asp
+ */
+const SEARCH_INPUT = document.getElementById('search');
+const ITEM_CLASS = 'autocomplete-items';
+const ACTIVE_CLASS = 'autocomplete-active';
+const LIST_ID = 'autocomplete-list';
```

Comment:
There are two lists in this file the list of auto-complete suggestions and the list of selected colleges. Use a more specific variable name to clarify which one this is.

Reply:
done!